lambda/README.md

```diff
@@ -9,14 +9,16 @@ Used for managing lambda functions.
 module "lambda" {
   source = "github.com/dbl-works/terraform//lambda?ref=main"
 
-  project     = "dbl"
-  environment = "production"
-  source_dir  = "Path to the directory containing the lambda function code."
+  function_name   = "lambda"
+  project         = "dbl"
+  environment     = "production"
+  source_dir = "Path to the directory containing the lambda function code."
 
   # optional
   handler     = "index.handler"
   timeout     = 10
   memory_size = 1024
+  runtime     = "ruby3.2"
```

Comment:
⚠️ default value in the variables files is `nodejs16.x`. We should have the same value in the README and the defaults.